packages/gestalt/src/SideNavigation/getChildrenToArray.js

```diff
@@ -17,7 +17,6 @@ const getChildrenToArray = ({
   const navigationChildren = [];
 
   let recursionLevel = 0;
-
```

Comment:
this is irrelevant to the PR, should be reverted

Reply:
DONE